docs/src/content/docs/guides/components.mdx

```diff
@@ -87,6 +87,49 @@ The code above generates the following tabs on the page:
 	</TabItem>
 </Tabs>
 
+#### Synced tabs
+
+Keep multiple sets of tabs on the same page in sync with the `syncKey` attribute.
+Switching to a synced tab will also switch to the tab with the same label in other `<Tabs>` that share the same `syncKey`.
+
+To synchronize tabs, add an identical `syncKey` property to each related `<Tabs>` component.
```

Comment:
```suggestion
Update the display of multiple tab groups at the same time by adding the `syncKey` attribute.

All `<Tabs>` on a page with the same `syncKey` value will display the same active label. This allows your reader to choose once (e.g. their operating system or package manager), and and see their choice reflected throughout the page.

To synchronize related tabs, add an identical `syncKey` property to each `<Tabs>` component and ensure that they all use the same `<TabItem>` labels:
```
Here's my attempt? Spark any better idea?

Reply:
I like this!

I also still quite like “Keep […] in sync” because it’s not so much the author who will “update the display,” more how they are managing the user’s experience.

Maybe we can keep it with something like this?

```suggestion
Keep multiple tab groups synchronized by adding the `syncKey` attribute.

All `<Tabs>` on a page with the same `syncKey` value will display the same active label. This allows your reader to choose once (e.g. their operating system or package manager), and see their choice reflected throughout the page.

To synchronize related tabs, add an identical `syncKey` property to each `<Tabs>` component and ensure that they all use the same `<TabItem>` labels:
```